language/use-cases/applying-llms-to-data/bigquery_dataframes_ml_drug_name_generation.ipynb

```diff
@@ -0,0 +1,989 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    prompt_generic = df_missing["openfda_generic_name"][k].iloc[0]
</pre><p>This lines throws an error when I execute it, I think because there is no index of 0 in the "df_missing" dataframe by this point. I think this is because the df gets filtered in previous steps, and the indices remain from the original (unfiltered) version.</p><p> </p><p>This can be fixed by adding a "reset_index" to when df_missing is last filtered, in this step above...</p><p> </p><p>df_missing = df_missing.head(100).reset_index(drop = True)</p><p> </p><p>...and then removing the 3 ".iloc[0]" references in this cell.</p><p> </p><p>prompt_generic = df_missing["openfda_generic_name"][k]</p><p>prompt_usage = df_missing["indications_and_usage"][k]</p><p>response_str = response[k]</p><p> </p><p>When I make those changes, the code runs as expected for me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
Done.